test/cpp/tensorexpr/test_loopnest.cpp

```diff
@@ -3921,5 +3921,29 @@ TEST(LoopNest, DISABLED_ColReduceSplitMaskUnevenReorder) {
   checkColReduce(s, *p.first, p.second);
 }
 
+TEST(LoopNest, DISABLED_VectorizeUse) {
+  KernelScope kernel_scope;
+  constexpr int N = 8;
+  Placeholder a("a", kFloat, {N});
+  Tensor* b = Compute(
+      "b", {{N, "n"}}, [&](const VarHandle& n) { return a.load(n) + 1.0f; });
+  Tensor* c = Compute(
+      "c", {{N, "n"}}, [&](const VarHandle& n) { return b->call(n) + 2.0f; });
+  LoopNest nest({c});
+  auto loops = nest.getLoopStmtsFor(b);
+  nest.vectorize(loops[0]);
+  loops = nest.getLoopStmtsFor(c);
+  nest.vectorize(loops[0]);
```

Comment:
I don't think this case ever happens in the regular flow in NNC because we "inline" all such calls before vectorization. See the call here: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/tensorexpr/kernel.cpp#L1567

The argument then is - are we doing the right thing? So far, we have been inlining everything, so this would never happen. But may be needed going forward.

Are you seeing real use of such flows (where we don't inlining the intermediate tensors)?

Reply:
Yeah the real use case here is to support convolution with padding.  This isn't the only way to do it but essentially I want to:
```
auto paddedInput = Compute(...);
auto conv = Compute(/* use paddedInput */);
// Apply some scheduling primitives to conv
```